core/src/main/java/com/gentics/mesh/cli/AbstractBootstrapInitializer.java

```diff
@@ -1017,4 +1018,12 @@ public BinaryDao binaryDao() {
 	public S3BinaryDao s3binaryDao() {
 		return daoCollection.s3binaryDao();
 	}
+
```

Comment:
This method is created specifically for doc generator, to keep its incompletely initialized Mesh working.